prow/plugins.yaml

```diff
@@ -17,6 +17,10 @@ triggers:
   - containerd/cri-containerd
   trusted_org: containerd
   join_org_url: "https://github.com/containerd/containerd/blob/master/MAINTAINERS"
+- repos:
+  - kubernetes-sig-testing
+  trusted_org: kubernetes
+  join_org_url: "https://git.k8s.io/community/community-membership.md#member"
```

Comment:
This doesn't need to be its own trusted org entry. `kubernetes-sig-testing` can just be included in the list of repos that have `kubernetes` as the trusted org.  I don't think there is a benefit in separating out this org from the others like there is for the Tide or approve config.

Reply:
ok this makes way more sense now, thanks